.gitlab-ci.yml

```diff
@@ -0,0 +1,31 @@
+# Cache gems in between builds
```

Comment:
Hi @tpitale, this was there because my initial fork was running here: https://gitlab.com/brodock/mail_room/tree/sentinel-support so I needed that file to run it with our CI.

It should do no harm, but I can remove it from the commit if you want. (as a side-effect, anyone who clone / fork using GitLab gets CI enabled automatically if the file stays)